src/localization/i18n.js

```diff
@@ -0,0 +1,35 @@
+const _ = require('lodash');
+
+const Utils = require('../utils/utils');
+const logger = require('../utils/logger');
+const locale = require('./locale.json');
+
+class i18n {
+  constructor() {
+    // Set locale based on OS language
+    let lang = 'en';
```

Comment:
the default is `zh` according to frontend

Reply:
done